Source/WebCore/page/RemoteFrame.cpp

```diff
@@ -150,4 +150,9 @@ void RemoteFrame::documentURLForConsoleLog(CompletionHandler<void(const URL&)>&&
     m_client->documentURLForConsoleLog(WTFMove(completionHandler));
 }
 
+void RemoteFrame::loadFrameRequest(FrameLoadRequest&& request)
```

Comment:
This function already exists: `RemoteFrame::changeLocation`.

Reply:
oops, updated!